Tests/ReactiveSwiftTests/SignalSpec.swift

```diff
@@ -2529,6 +2529,78 @@ class SignalSpec: QuickSpec {
 			}
 		}
 
+		describe("timeout(after:sending:on:)") {
+			var testScheduler: TestScheduler!
+			var signal: Signal<Int, TestError>!
+			var observer: Signal<Int, TestError>.Observer!
+
+			beforeEach {
+				testScheduler = TestScheduler()
+				let (baseSignal, incomingObserver) = Signal<Int, TestError>.pipe()
+				signal = baseSignal.timeout(after: 2, sending: .value(-1), on: testScheduler)
+				observer = incomingObserver
+			}
+
+			it("should complete if within the interval") {
```

Comment:
Could you please test also sending `interrupted`, and whether the value is really delivered when `sending: .value(...)` is used?